DistributionPackages/Neos.NeosIo.ServiceOfferings/Resources/Private/Fusion/Documents/ServiceProvider.fusion

```diff
@@ -1,6 +1,10 @@
 prototype(Neos.NeosIo.ServiceOfferings:Document.ServiceProvider) < prototype(Neos.NeosIo:DefaultPage) {
     body {
         content {
+            keyVisual {
+                keyVisualImage = ${q(documentNode).property('bannerImage')}
```

Comment:
What happens if no bannerImage is set? Falls back automatically to default?

Reply:
Yes, just did an override of the property name, everything else is the same.